strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java

```diff
@@ -90,15 +95,15 @@ public RepositoryPath getName(int index)
         return wrap(getTarget().getName(index));
     }
 
-    public Path subpath(int beginIndex,
-                        int endIndex)
+    public RepositoryPath subpath(int beginIndex,
```

Comment:
@sbespalov : Would you mind renaming this to `subPath` in a follow-up pull?

Reply:
it's overwritten method, so I can not rename it :)